### ISSUE TITLE
Is there a Plaid Link flow for users to directly update their MFA credentials?

### ISSUE THREAD

**Issue by guillelopez** [plaid/link#37]
Hi. Does Plaid Link provide any functionality for when a connection MFA access has changed or application's access has been revoked? I am wondering what's the best way to tackle this problem. Thanks.

**Comment by michaelckelly** [comment]
@guillelopez -

Link does not currently support the [update user](https://plaid.com/docs/#update-auth-user)/PATCH flow. You'll need to implement that logic yourself for the time being. I do realize it'd be ideal if Link could handle that for you - it's a feature we'd like to add in the future, but we do not have a firm timeline for that yet.

**Comment by seanmccann** [comment]
I wish I would have read this before starting to implement Link... Without updating functionality, Link is useless for apps that have real world users that change their passwords/MFA.

**Comment by pcheek** [comment]
^ +1

**Comment by michaelckelly** [comment]
@seanmccann @pcheek 

I completely understand your frustration around this issue. We're working hard to add PATCH functionality to Link and expect to have news on that front within the next few weeks. We'll be sure to update this issue with progress, too!

**Comment by pcheek** [comment]
Awesome, thanks!

**Comment by seanmccann** [comment]
Cool, looking forward to it! Thanks for the update!

**Comment by michaelckelly** [comment]
@guillelopez @seanmccann @pcheek - 

Check out the new [updating existing user accounts](https://github.com/plaid/link#updating-existing-user-accounts) section in the Link docs!

**Comment by guillelopez** [comment]
That's great @michaelckelly. Thanks for taking care of this.

**Comment by michaelckelly** [comment]
I'm going to ahead and close those - feel free to open a [new issue](https://github.com/plaid/link/issues/new) if you have any questions on Link's update user functionality!
